teaser/logic/buildingobjects/useconditions.py

```diff
@@ -176,6 +176,9 @@ class UseConditions(object):
         false = natural infiltration + ventilation due to a AHU
         + window infiltration calculated by window opening model
         AixLib: Used on Zone level for ventilation.
+    static_infiltration: float [1/h]
+        Infiltration rate for static heat load calculation.
+        Default is 0.5 based on the DIN EN 12831-1:2017 minimal air exchange rate reference value.
```

Comment:
Mhm, this is now only used to calculate the heat load, or? Maybe specify "normative_infiltration"? I found it confusing at first. Also, maybe raise a warning during heat load calculation if base_infiltration is larger than 0.5, which could lead to ideal heaters being too small.